class.jetpack-onboarding-end-points.php

```diff
@@ -539,6 +545,41 @@ static function install_theme() {
 		wp_send_json_success( $theme_id );
 	}
 
+	static function install_plugin( $slug ) {
+		if ( is_multisite() && ! current_user_can( 'manage_network' ) ) {
```

Comment:
I think only folks of `admin` or higher will see the onboarding process, but it still may be worth adding in more permission checks.